client/landing/stepper/declarative-flow/helpers/test/index.ts

```diff
@@ -0,0 +1,29 @@
+import { shouldRedirectToSiteMigration } from '..';
+import { STEPS } from '../../internals/steps';
+describe( 'DeclarativeFlowHelpers', () => {
+	it( 'returns true when current step, platform and origin as set properly', () => {
+		expect(
+			shouldRedirectToSiteMigration(
+				STEPS.IMPORT_LIST.slug,
+				'wordpress',
+				'site-migration-identify'
```

Comment:
Should we use the constant here instead of the raw string?
```suggestion
				STEPS.SITE_MIGRATION_IDENTIFY.slug
```

Reply:
Fixed here 9cc65c949135b3e23f0ec252bb634306d54d7ed8